.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Not relevant for this project since it doesn't contain code. Either we exclude all editor specific files, or none.

```suggestion
```